Sources/SWBCore/DependencyResolution.swift

```diff
@@ -742,8 +743,9 @@ extension SpecializationParameters {
                             continue
                         }
 
+                        let behavior: Diagnostic.Behavior = buildRequest.enableIndexBuildArena ? .warning : .error
```

Comment:
@bnbarham I would expect that we may encounter this error more often after my change and we do treat similar errors as just warnings for indexing, so I opted to change this here as well preemptively.